### DISCUSSION TITLE
how to remove symbol from layer?

### DISCUSSION THREAD

**Discussion by ishafiul** [maplibre/flutter-maplibre-gl#497] [Q&A]
Hello, everyone. I'm using MapLibre Flutter for my application. To display map symbols, I’ve added a source with a feature layer. Now, I want to remove a specific marker symbol from the map. I’ve tried using `controller.symbolManager?.byId` and then calling `controller.symbolManager.remove`, but it seems like `controller.symbolManager?.byId` is returning a null symbol. How can I achieve this? I need help.

**Comment by josxha** [comment]
There definitely is something odd about byId(). When calling `addSymbol()` the function should return the representating Symbol object with its id. Try to store it and use it to remove the symbol from the map.

**Reply by ishafiul** [reply]
Is anyone working on this, or is there any way to archive it?
